### PR TITLE
Dustplacer Feature

### PR DESCRIPTION
**Pull request by Ninty9**
Basic functionality for the feature is done

### CODE REVIEW

File: src/main/java/com/domain/redstonetools/features/commands/DustPlacerFeature.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    static public Boolean isActive = false;
+    static public List<Identifier> blockList = new ArrayList<>();
+
+    public boolean blockInBlockList(Identifier block) {
+        return false;
+    }
+
+    @Override
+    protected int execute(ServerCommandSource source) throws CommandSyntaxException {
+        if(blockList.isEmpty()) {
```
Comment (MiranCZ) [inline src/main/java/com/domain/redstonetools/features/commands/DustPlacerFeature.java:29]: maybe add a `static{` statement to add all those values into list

Comment (Ninty9) [reply src/main/java/com/domain/redstonetools/features/commands/DustPlacerFeature.java]: Good idea, added. (also cleaned up a method that was lying around for some reason, don't know why i overlooked that.)

**Review by Matthias1590 — CHANGES_REQUESTED** [review]
The list of blocks should be stored in a settings file, the toggling of the feature should be done using 2 commands: /autodust enable and /autodust disable (override the registerCommands method to register them)

**Comment by Matthias1590** [comment]
any updates on this?

**Comment by orbyfied** [comment]
> The list of blocks should be stored in a settings file (use dependency injection to inject something like a config provider or config reader into the autodust feature), the toggling of the feature should be done using 2 commands: /autodust enable and /autodust disable (override the registerCommands method to register them)

I think we should first get a configuration system working.

**Comment by Matthias1590** [comment]
> > The list of blocks should be stored in a settings file (use dependency injection to inject something like a config provider or config reader into the autodust feature), the toggling of the feature should be done using 2 commands: /autodust enable and /autodust disable (override the registerCommands method to register them)
> 
> I think we should first get a configuration system working.

that would be the configuration system

**Comment by MiranCZ** [comment]
Why was this closed?

**Pull request closed without merging**